src/unpackertool/plugins/mpress/mpress.h

```diff
@@ -7,12 +7,16 @@
 #ifndef UNPACKERTOOL_PLUGINS_MPRESS_MPRESS_H
 #define UNPACKERTOOL_PLUGINS_MPRESS_MPRESS_H
 
+#include <pelib/PeLib.h>
+
 #include "retdec/loader/loader.h"
-#include "retdec/unpacker/dynamic_buffer.h"
+#include "retdec/utils/dynamic_buffer.h"
 #include "retdec/unpacker/plugin.h"
 
 #define mpress_plugin plugin(retdec::unpackertool::mpress::MpressPlugin)
 
+using namespace retdec::utils;
```

Comment:
Same as the comment I made earlier. Never put `using namespace` in header files.